dataverse-webapp/src/main/webapp/notificationTemplate.xhtml

```diff
@@ -0,0 +1,9 @@
+<ui:composition
+        xmlns:ui="http://java.sun.com/jsf/facelets"
+>
+
+
+
```

Comment:
Maybe some comment in `ui:remove` tag why it is here?

Reply:
You mean ui:composition? I will add the comment.

Reply:
I meant:
```
<ui:remove>
<!-- comment -->
</ui:remove>
```
That way it will not be rendered in final response. But I guess that nothing bad would happen if we would add simple `<!-- comment -->`